### ISSUE TITLE
update instructor notes Zoom section 

### ISSUE THREAD

**Issue by brownsarahm** [carpentries/instructor-training#1122]
since the handbook zoom setting is more up to date the instructor notes should refer to that maybe?

**Comment by maneesha** [comment]
I'm assuming you are referring to [Section IV here](https://carpentries.github.io/instructor-training/guide/index.html#zoom).

Looking at that, it seems like there is a mix of information on how to use Zoom and how to manage an event in general.  It may make sense to seperate out the general information, keeping it here. The Zoom specific information can stay in the handbook.  See https://docs.carpentries.org/topic_folders/communications/tools/zoom_rooms.html.

It also looks like it could use some general edits (like the line "Suggestion from AN....").

**Comment by brownsarahm** [comment]
yes, keeping the general parts in instructor notes and zoom-specific to handbook with link seems like the right path.

**Comment by kcranston** [comment]
I think that subsequent changes to the lesson have rendered this issue no longer relevant. For example, 

> I'm assuming you are referring to [Section IV here](https://carpentries.github.io/instructor-training/guide/index.html#zoom).

This page no longer exists in the lesson.
